src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -2533,6 +2535,8 @@ private bool IsEndOfNameInExplicitInterface()
             return this.CurrentToken.Kind == SyntaxKind.DotToken || this.CurrentToken.Kind == SyntaxKind.ColonColonToken;
         }
 
+        private bool IsEndOfFunctionPointerParameterList() => this.CurrentToken.Kind == SyntaxKind.GreaterThanToken;
```

Comment:
>this.CurrentToken.Kind == SyntaxKind.GreaterThanToken [](start = 62, length = 53)

Should we also recognize ```OpenParenToken```?

Reply:
Yes, but only the case where we started with an `OpenParenToken`.

---
In reply to: [349368015](https://github.com/dotnet/roslyn/pull/39879#discussion_r349368015) [](ancestors = 349368015)